src/pageObject.js

```diff
@@ -2,14 +2,38 @@
 
 class Header {
 
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
```

Comment:
Here, you are creating a new instance of `Header` in the constructor of `PageObject`. This tightly couples these two classes together. It's generally better to pass in instances of dependencies to the constructor. This is called dependency injection and it makes your code more flexible and easier to test.